auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -379,100 +249,8 @@ export default defineComponent({
-
-      let filingResponse = null
-      if (regTypes.includes(business.nameRequest?.legalType)) {
-        filingResponse = await businessStore.createNamedBusiness({ filingType: FilingTypes.REGISTRATION, business })
-      } else if (iaTypes.includes(business.nameRequest?.legalType)) {
-        filingResponse = await businessStore.createNamedBusiness({
-          filingType: FilingTypes.INCORPORATION_APPLICATION, business })
-      }
-
-      if (filingResponse?.errorMsg) {
-        context.emit('add-unknown-error')
-        return ''
-      }
-      return filingResponse.data.filing.business.identifier
-    }
-
-    /** Navigation handler for entities dashboard. */
-    const goToDashboard = (businessIdentifier: string): void => {
-      ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, businessIdentifier)
-      let redirectURL = `${ConfigHelper.getBusinessURL()}${businessIdentifier}`
-      window.location.href = appendAccountId(decodeURIComponent(redirectURL))
-    }
-
-    /** Navigation handler for Name Request application. */
-    const goToNameRequest = (nameRequest: NameRequest): void => {
-      ConfigHelper.setNrCredentials(nameRequest)
-      window.location.href = appendAccountId(`${ConfigHelper.getNameRequestUrl()}nr/${nameRequest.id}`)
-    }
-
-    /** Handler for open action */
-    const open = (item: Business): void => {
-      if ((item.corpType?.code || item.corpType) === CorpTypes.NAME_REQUEST) {
-        goToNameRequest(item.nameRequest)
-      } else {
-        goToDashboard(item.businessIdentifier)
-      }
-    }
-
-    /** Navigation handler for OneStop application */
-    const goToOneStop = (): void => {
-      window.location.href = appendAccountId(ConfigHelper.getOneStopUrl())
-    }
-
-    /** Navigation handler for Corporate Online application */
-    const goToCorpOnline = (): void => {
-      window.open(appendAccountId(ConfigHelper.getCorporateOnlineUrl()), '_blank')
-    }
-
-    /** Navigation handler for Societies Online */
-    const goToSocieties = (): void => {
-      window.open(appendAccountId(ConfigHelper.getSocietiesUrl()), '_blank')
-    }
-
-    /** Handler for draft IA creation and navigation */
-    const useNameRequest = async (item: Business) => {
-      switch (item.nameRequest.target) {
-        case NrTargetTypes.LEAR: {
-          // Create new IA if the selected item is Name Request
-          let businessIdentifier = item.businessIdentifier
-          if (item.corpType.code === CorpTypes.NAME_REQUEST) {
-            businessIdentifier = await createBusinessRecord(item)
-          }
-          goToDashboard(businessIdentifier)
-          break
-        }
-        case NrTargetTypes.ONESTOP:
-          goToOneStop()
-          break
-        case NrTargetTypes.COLIN:
-          goToCorpOnline()
-          break
-      }
-    }
-
-    /** Emit business/nr information to be unaffiliated. */
-    const removeBusiness = (business: Business): RemoveBusinessPayload => {
```

Comment:
Remove buttons wont work without this part of the code.
You will need to emit it from the AffiliationAction, then catch it here and pass it up one level.

There is function you can take from my PR that does it.